backend/app/schemas.py

```diff
@@ -1,11 +1,18 @@
 import datetime
+from typing import Optional
 
 from pydantic import BaseModel
 
-from app.enums import TransactionCategory
+from app.enums import TransactionCategory, TransactionType
+
+
+class Balance(BaseModel):
+    value: float
+    currency: str
 
 
 class Transaction(BaseModel):
+    id: Optional[int]
```

Comment:
We might use uuid. INT in sql is limited.
e.g `id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)`

Reply:
Let's create a ticket for this. It's low priority to update it now.

Reply:
Good point.